### ISSUE TITLE
Zoom To Feature Sort Alphabetically

### ISSUE THREAD

**Issue by kcarrier** [tmcgee/cmv-widgets#48]
Is it possible to sort attributes from a field alphanumerically?

**Comment by kcarrier** [comment]
I tried adding the orderbyFields but the results are still not in alphabetical order.

```
        getFeatures: function () {
            var query = new Query();
            query.outFields = [this.field];
            query.orderByFields = [this.field];
            query.where = this.where;
            query.returnGeometry = true;
            //query.returnDistinctValues = true;
            query.outSpatialReference = {
                wkid: this.spatialReference
            };

            var queryTask = new QueryTask(this.url);
            queryTask.execute(query, lang.hitch(this, 'populateList'));
        },
```

**Comment by kcarrier** [comment]
I forgot to hit the save button, adding orderByFields is now working.

**Comment by tmcgee** [comment]
Thanks @kcarrier I had that same line in an enhancement made for a client that had not made it back to the repo. Adding it now.

**Comment by kcarrier** [comment]
@tmcgee - FYI, I was working with a client who was still at 9.3 and according to Esri documentation this will only work when the REST endpoint is 10.1 or greater. They published a 10.3.1 service and it loaded no problem. The only other question pertaining to this widget would be when it loads. The initial query for me is taking some time 2-4 seconds, is there a way to initiate the query after the cmv application loads so when the user goes to activate the widget the query has already occurred and the drop down is already populated?
